keyboards/handwired/derb_planck/rev1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,117 @@
+        _______, _______, _______, _______, _______, _______, _______, _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+    ),
+
+    /* Raise
+     * ,-----------------------------------------------------------------------------------.
+     * |   `  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Bksp |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * | Del  |M_left|M_down| M_up |M_righ|      |      |   -  |   =  |   [  |   ]  |  \   |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |M_Btn2|M_Btn1|  F11 |  F12 |ISO # |ISO / |Pg Up |Pg Dn |      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      | Next | Vol- | Vol+ | Play |
+     * `-----------------------------------------------------------------------------------'
+     */
+    [_RAISE] = LAYOUT_ortho_4x12(
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+        KC_DEL,  KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R, _______, _______, KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+        _______, _______, _______, KC_BTN2, KC_BTN1, KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
+    ),
+
+    /* Adjust (Lower + Raise)
+     * ,-----------------------------------------------------------------------------------.
+     * |      |      |      |      |      |      |      |      |      |      |      |  Del |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |USB3_P|USB2_P|USB1_P|      |      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |      |      |      |      |      |      |      |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      | Reset|      |      |      |      |      |      |      |      |      |      |
+     * `-----------------------------------------------------------------------------------'
+     */
+    [_ADJUST] = LAYOUT_ortho_4x12(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL ,
+        _______, _______, _______, _______, _______, _______, _______, USB3,    USB2,    USB1,    _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
This is meant for usb selection, right? 

If so, you should probably move this to the keyboard level, then.

Reply:
moved to rev1.c